### PR TITLE
Add user signup e2e tests and travis config

### LINKED ISSUE #101: Include E2E testing to CI pipeline

### PR DESCRIPTION
**Pull request by saw-jan**
## Description

Added e2e test for different scenarios involved in signing up a new user.

## Test Scenarios 🗒
* Valid sign up
* Sign up with invalid email
* Sign up with unmatched password
* sign up with empty fields
* Sign up with already registered email
* Go to login page from sign up page

### Also ⚙
 Added travis configuration to run e2e tests in travis CI 
closes #101

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,36 @@
-language: ruby
+laanguage: ruby
```
Comment (individual-it) [inline .travis.yml:1]: random extra `a`

**Comment by koushikmohan1996** [comment]
@individual-it Thanks for the review

@saw-jan Everything works now, you are awesome. Can I merge the PR now?

**Comment by saw-jan** [comment]
@koushikmohan1996 yeah, sure. u may

**Comment by koushikmohan1996** [comment]
@all-contributors add @saw-jan for test

**Pull request merged**